proposals/texel-buffers.md

```diff
@@ -399,3 +402,8 @@ R32G32B32A32_FLOAT
        - Make it core.
        - Drop the formats that are not widespread (leaving them for a [future texture format tier extension](https://github.com/gpuweb/gpuweb/issues/3837)).
        - We do not need to support Metal <2.1 (Metal 2.2 is our minimum requirement now).
+2. In the original issue it was mentioned [#162 (comment)](https://github.com/gpuweb/gpuweb/issues/162#issuecomment-452771668)
+   that uniform texel buffers support more formats. This proposal is only for storage texel buffers but uses the name "texture buffer" throughout.
+
+   - Is it worth adding uniform texel buffers? Besides wider format support, are they faster?
+   - If the answer is yes or not sure, we should probably use "storage texture/texel buffer" for this proposal instead.
```

Comment:
We talked a bit about this at the F2F and I realized the intent was to allow implementations to translate read-only texel buffers to uniform texel buffers which makes sense and unifies things nicely. The texture format table can then enable usage of more formats for read-only texel buffers.

From my side, I consider this resolved but if others are still concerned about this we can keep it.

Reply:
I'll update this PR with that resolution.